scripts/get_weights.py

```diff
@@ -0,0 +1,94 @@
+import torch
+from safetensors.torch import save_file
+import urllib.request
+import os
+import sys
+import argparse
+
+data_path = os.path.join(os.path.dirname(__file__), "../data/")
+
+def ensure_data_dir():
+    print("Ensuring empty data directory...")
+    if not os.path.exists(data_path):
+        os.mkdir(data_path)
+        print("Created empty data directory")
+        return
+    else:
+        print("Found data directory")
+
+    if os.listdir(data_path):
+        while (response := input("Data directory isn't empty. Would you like to overwrite its contents? [(y)/n]")) not in ["y","n",""]:

```

Comment:
I would rather just erroring out and tell the user to rm the directory in this case rather than having the interactive version to handle (which might not work well if incorporated in another shell script etc), also calling `os.remove` automatically seems potentially scary. What do you think?

Reply:
Hmm I think I see what you mean, another case could be if someone trains their own stable diffusion model and wants to use it, then deleting it by accident would be pretty easy with this tool. I'll change it to make sure only the files it's about to download aren't present, otherwise it will just fail.